libs/design-system/src/lib/Pages/HomePage/index.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  LandingHeaderHome,
+  SocialLinksSignup,
+  Footer,
+  LinkCard2Column,
+  LinkCard3Column,
+  Flips,
+  ToolsAndConcepts,
+  UpcomingEvents,
+} from '../../Components';
+import { FlipCellProps } from '../../Components/FlipCell';
+import { ToolCardProps } from '../../Components/ToolCard';
+import { LinkCard3ColumnProps } from '../../Components/LinkCard3Column';
+import { LinkCard2ColumnProps } from '../../Components/LinkCard2Column';
+import { UpcomingEventsProps } from '../../Components/UpcomingEvents';
+
+export type HomePageProps = {
+  startProjectItems: LinkCard2ColumnProps;
+  flips: FlipCellProps[];
+  tools: ToolCardProps[];
+  threeColumnItems: LinkCard3ColumnProps;
+  upcomingEvents: UpcomingEventsProps;
+};
+
+const HomePage = ({
+  startProjectItems,
+  flips,
+  tools,
+  threeColumnItems,
+  upcomingEvents,
+}: HomePageProps) => {
+  return (
+    <div className="bg-primary-gray-50 dark:bg-black">
+      <LandingHeaderHome
+        title="Developer Resources"
+        description="Understand the foundational concepts of Flow and its language, Cadence"
+        tag="onflow"
+      />
+
+      <LinkCard2Column {...startProjectItems} bottomRounded={false} />
+
+      <div className="mb-36">
```

Comment:
The new PageSections and PageSection components could be used here for spacing and horizontal borders

Reply:
Doesn't look like there's borders on the home page or am I crazy?